source/about/mobile-app-changelog.md

```diff
@@ -9,6 +9,33 @@ This changelog summarizes updates to Mattermost mobile apps releases for [Matter
 ```{include} common-esr-support.md
 ```
 
+(release-v2-27-0)=
+## 2.27.0 Release
+ - Release Date: April 16, 2025
+ - Server Versions Supported: Server v9.11.0+ is required. Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device.
+
+### Compatibility
+ - **Upgrade to server version v9.11.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/about/release-policy.html#extended-support-releases) (ESR) v9.5.0 has ended and upgrading to server ESR v9.11.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 6s devices and later with iOS 15.1+ are required.
+
+### Improvements
+ - Added the ability to configure per server (Enterprise plan) prompting for biometric authentication, preventing the use of the app if the device is jailbroken / rooted and preventing screen captures. 
+ - Updated empty state images to align with new branding style. 
+ - Added the ability to play and download audios on the chat. 
+ - Added custom profile attributes to the edit profile screen. 
```

Comment:
Are the custom profile attributes functional as of this release?

Reply:
It was released in v10.5 https://mattermost.atlassian.net/browse/MM-59498, so I believe so, but I'll let the Workflows team confirm. cc @sbishel @mgdelacroix @Willyfrog

Reply:
I think it still needs to be enabled via feature flag,  as it was before